test/test_integration.py

```diff
@@ -1076,6 +1076,8 @@ def test_tabs_with_children_undefined(self):
 
         self.startServer(app=app)
 
+        self.wait_for_element_by_css_selector('#tabs-content')
```

Comment:
There was a percy diff - the image produced in this PR prior to this commit was actually the right one, dunno how the incorrect one got approved... but to make that more robust I added this wait for the correct element to be on the page.